ci/code_checks.sh

```diff
@@ -266,7 +266,12 @@ if mods:
     sys.exit(len(mods))
     "
     RET=$(($RET + $?)) ; echo $MSG "DONE"
-
+	
+	MSG='Check code for instances of os.remove' ; echo $MSG
```

Comment:
this needs to be lined up with the others

you can do a grep check similar to others